test/spec/components/TextField.spec.js

```diff
@@ -101,8 +102,14 @@ describe('<TextField>', function() {
   it('should have no trailing spaces', async function() {
 
     // given
-    const result = createTextField({ container });
+    const flushSpy = sinon.spy();
```

Comment:
To test:

* On blur, we trim
* With a pending input (debounce) we commit on blur